library/db/src/main/kotlin/org/cru/godtools/db/room/entity/partial/SyncLanguage.kt

```diff
@@ -5,5 +5,6 @@ import org.cru.godtools.model.Language
 class SyncLanguage(language: Language) {
     val code = language.code
     val name = language.name
+    val forceLanguageName = language.forceLanguageName
```

Comment:
@frett would you like this changed to `isForcedName` as well?

Reply:
yeah, it will have to be in order for the partial entity to work correctly